### ISSUE TITLE
CPVs wrongly formatted in actor details query

### ISSUE THREAD

**Issue by zufanka** [tenders-exposed/elvis-ember#306]
(possibly backend issue? @georgiana-b )

![image](https://user-images.githubusercontent.com/2411066/30843876-3d83087e-a28c-11e7-95ef-69fff0ceb1d7.png)

![image](https://user-images.githubusercontent.com/2411066/30843874-3738caa8-a28c-11e7-9b42-edc93803783f.png)

[example network](https://elvis-ember-develop.herokuapp.com/network/591dc03a2dcb565324000011/procurers)

**Comment by georgiana-b** [comment]
Actually, this is not related to contracts with 0 sum but it certainly happens, although I can't see any pattern yet.

For example [this procurer]( http://tenders.exposed/network/59cb39fa2dcb565b0300007a/procurers/29074) is linked only to one suppliers with sum 0 but the others, including those with greater sums are not shown.

The payload of the network doesn't show all the nodes returned by procuring entity details. This is a backend issue so I opened tenders-exposed/elvis-backend#65 and I'll close this.

**Comment by georgiana-b** [comment]
Ok, turns out the network was ok but the actor details were showing all the contracts of the actor because the filters (years, cpvs etc.) weren't applied. I fixed that in tenders-exposed/elvis-backend#67,  deployed and tested it so now the backend works as expected on production.

However, I have to reopen this because currently the [procurer details page](http://tenders.exposed/network/59cb39fa2dcb565b0300007a/procurers/29074) shows no contracts. This is because the query sent to the backend on `/api/v1/contracts/procuring_entities_details` is wrong:

```
{
        "query": {
          "procuring_entities": [29074],
          "years": [2001,2002,2003,2004,2005,2006,2007,2008,2009,2010,2011],
          "countries": ["UK"],
          "cpvs": ["72000000,72200000,72212190,72212222,72212311,72220000,72221000,72222000,72222300,72224000,72225000,72242000,72260000,72261000,72262000,72267100,72268000,72310000,72314000,72315000,72320000,72410000,72412000,72500000,72610000,72710000"]
        }
      }
```

Notice the formatting mistake in the array of `cpvs`.  It should be an array with each CPV as a separate string.
